ansible/roles/ocp4_workload_rhacm/tasks/pre_workload.yml

```diff
@@ -0,0 +1,48 @@
+---
+# Implement your Pre Workload deployment tasks here
+- set_fact:
+    ansible_python_interpreter: "/bin/python"
+
+- name: Setup pre-requisite python3 packages
+  package:
+    state: present
+    name:
+    - git
+    - python3-pip
+  become: true
+
+- name: Setup pre-requisite pip3 packages
+  pip:
+    name:
+    - virtualenv
+    state: present
+    executable: /usr/bin/pip3
+  become: true
+
+- name: Copy requirements.txt to target for k8s virtualenv
+  copy:
+    src: ./files/requirements_k8s.txt
+    dest: /tmp/requirements_k8s.txt
+  become: true
+
+- name: "Create virtualenv k8s"
+  pip:
+    requirements: /tmp/requirements_k8s.txt
+    virtualenv: "/opt/virtualenvs/k8s"
```

Comment:
And this will be a no-op.... except your requirements may override the ones already there.

Reply:
Changed.